Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
```

Comment:
Question on a different note: Is Point *p supposed to be modified in function InsertPoint? If not, we should take the habbit of const whenever possible: "Point const *" or "const Point *" (or alternatives without bare pointers).